### PR TITLE
Fix: src: device: manager: Panic when call unimplemented options

### PR DESCRIPTION
**Pull request by RaulTrombin**

### CODE REVIEW

File: src/device/manager.rs [hunk 1]
```diff
@@ -174,7 +175,14 @@ impl DeviceManager {
                     );
                 }
             }
-            _ => todo!(), // Unreachable, DeviceManagerHandler uses GetDeviceHandler and forwards the requests.
+            _ => {
+                if let Err(e) = actor_request
+                    .respond_to
+                    .send(Err(ManagerError::NotImplemented(actor_request.request)))
+                {
+                    warn!("DeviceManager: Failed to return response: {:?}", e);
```
Comment (joaoantoniocardoso) [inline src/device/manager.rs:183]: We often prefer to use it like `{e:?}`, but I don't know if this repo is following this idea

**Review by joaoantoniocardoso — APPROVED** [review]
The code seems all right

File: src/device/manager.rs (hunk 1 above)
Comment (RaulTrombin) [reply src/device/manager.rs:183]: some habits are difficult to break, I learned this way when was learning C haha 
I'm going to apply and keep this stantandard then.

Comment (RaulTrombin) [reply src/device/manager.rs:183]: there some special cases that seems it doesn't work:

```
                    warn!(
                        "Unsupported request for device type: {:?}",
                        &self.device_type
                    );
```

Comment (RaulTrombin) [reply src/device/manager.rs]: ping @joaoantoniocardoso 
Please check the newer commit below!

**Review by joaoantoniocardoso — APPROVED** [review]

**Pull request merged**